thirdparty/kyaml/runfn/runfn.go

```diff
@@ -35,21 +35,13 @@ type RunFns struct {
 	// Path is the path to the directory containing functions
 	Path string
 
-	// FunctionPaths Paths allows functions to be specified outside the configuration
-	// directory.
-	// Functions provided on FunctionPaths are globally scoped.
-	// If FunctionPaths length is > 0, then NoFunctionsFromInput defaults to true
-	FunctionPaths []string
+	// FnConfigPath specifies a config file which contains the configs used in
```

Comment:
Specify the semantics of this string. Is it relative/absolute? Is it OS path or slash-separated?

Reply:
Updated the comment to 
```
// FnConfigPath specifies a config file which contains the configs used in
// function input. It can be absolute or relative to kpt working directory.
// The exact format depends on the OS.
```